authors/apps/authentication/serializers.py

```diff
@@ -1,4 +1,5 @@
 from django.contrib.auth import authenticate
+import re
```

Comment:
Import should be in this order - python's packages first, followed by the framework's packages, then third-party packages and then your own packages/modules